client/src/routes/fight/FightSideProposition.svelte

```diff
@@ -0,0 +1,45 @@
+<script>
+    import Button, {Label} from "@smui/button";
+
+    function handleClick(type){
+        console.log("clicked!")
+    }
+</script>
+
+<div class="sideDiv">
+    <p>Proponowane zdarzenia od sędziów bocznych:</p>
+    <Button class="redButton" disabled=true on:click={() => handleClick("red")}>
```

Comment:
Think about using `Function.bind(handleClick, "red")`, (but do not apply it just beacuse of this comment)

Reply:
What are the advantages of using binding here instead of arrow function?

Reply:
The only concern is readability, but again - it's up to you to decide.